src/Qir/Tests/QIR-dynamic/qsharp/qir-test-assert.qs

```diff
@@ -1,15 +1,235 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// https://docs.microsoft.com/en-us/azure/quantum/concepts-the-qubit#single-qubit-operations    Qubit and Operations
+// https://docs.microsoft.com/en-us/azure/quantum/user-guide/libraries/standard/prelude         Operation Details, Multi-Qubit Operations
+
 namespace Microsoft.Quantum.Testing.QIR  {
 
     open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    // |0> -> |1>
```

Comment:
In general, I'd suggest that in comments and strings, we use the right angle bracket character (`⟩`) in preference to the greater-than character (`>)`, since the former matches notation used in quantum computing.
```suggestion
    // |0⟩ -> |1⟩
```

Reply:
I would be glad to but not all the editors display/save those chars correctly. And the one that I use most often, does not :-(.

Also do you know the fast way to enter those chars from the keyboard other than copy-paste?
I know that their [codes](https://en.wikipedia.org/wiki/Bracket#Encoding_in_digital_media) are U+27E8, U+27E9. 
27E8, followed by <Alt+x>, only works in Word, but not in the code editors. Other [input methods](https://en.wikipedia.org/wiki/Unicode_input#Hexadecimal_input) are platform-dependent or require registry editing...

Reply:
For now I'm inclined to leave those as they are in the _non-doc_ comments, and to use your suggestion in the _doc-comments_.